### PR TITLE
[#2PlaysAMonth]: DevBlog project done

### LINKED ISSUE #987: [#2PlaysAMonth] devblog a dev to client :
### What's the Play about?

it's a dev.to client. custom designed home page. You can search and read dev to articles from here

### What ReactJS concept will be used to create this play?

React Hooks, States, Integrating API 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Coderamrin**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description
DevBlog play is done. it's the 2nd project for 2playsamonth

Fixes #987 

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [X ] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots or example output

### CODE REVIEW

File: src/plays/devblog/Devblog.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+// WARNING: Do not change the entry componenet name
+function Devblog(props) {
+  // Your Code Start below.
+
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={props} />
+        <div className="play-details-body">
```
Comment (atapas) [inline src/plays/devblog/Devblog.js:13]: @Coderamrin I have the same comment that given for the last play. Add a unique style and use it for all in your CSS file. Thanks!

File: src/plays/sticky-notes/App.js [hunk 2]
```diff
@@ -31,16 +31,18 @@ function App() {
...
-      ...note,
-      id: Math.random() * 10
-    };
-
-    setNotes([newNote, ...notes]);
-    setNote({
-      body: ''
-    });
-    setToggleForm(!toggleForm);
+    if (note.body) {
```
Comment (atapas) [inline src/plays/sticky-notes/App.js:34]: I think you need to pull your branch because this change is already part of the main.

**Review by atapas — CHANGES_REQUESTED** [review]
@Coderamrin review comments

File: src/plays/sticky-notes/App.js (hunk 2 above)
Comment (Coderamrin) [reply src/plays/sticky-notes/App.js]: @atapas fixed the bugs. check it out.

**Comment by atapas** [comment]
Please fix the compilation error before pushing

image

**Comment by Coderamrin** [comment]
image

fixed

File: src/plays/devblog/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+const Header = () => {
+  return (
+    <header className="flex bg-gray-800 text-white justify-between px-10 py-4 w-full">
+      <Link className="logo text-3xl font-bold" to="/plays/coderamrin/devblog">
+        DevBlog
+      </Link>
+
+      <nav>
+        <a href="https://amrin.onrender.com/">About me</a>
```
Comment (atapas) [inline src/plays/devblog/components/Header.js:11]: @Coderamrin Open this link in. anew tab.

**Review by atapas — CHANGES_REQUESTED** [review]
One small change + edit the play to add the recording link @Coderamrin

File: src/plays/devblog/components/Header.js (hunk 3 above)
Comment (Coderamrin) [reply src/plays/devblog/components/Header.js]: > @Coderamrin Open this link in a new tab.

@atapas  added the play video

**Review by atapas — APPROVED** [review]

**Pull request merged**